entities/src/main/java/com/flipkart/varadhi/entities/Subscription.java

```diff
@@ -2,7 +2,7 @@
 
 import jakarta.validation.constraints.NotNull;
 
-public class Subscription extends VaradhiResource {
+public class Subscription extends MetaStoreEntity {
```

Comment:
Why are we not using the validation annotation here?

Reply:
this will get removed in the subscription crud PR